doc/index.md

```diff
@@ -174,7 +174,7 @@ Now that you have LXD running on your system you can read the [getting started g
 Bug reports can be filed at: <https://github.com/lxc/lxd/issues/new>
 
 ## Contributing
-Fixes and new features are greatly appreciated but please read our [contributing guidelines](contributing.md) first.
+Fixes and new features are greatly appreciated but please read our [contributing guidelines](doc/contributing.md) first.
```

Comment:
This will break links for anyone looking at the documentation. Currently there are symlinks in place and a bug in Github's web UI where they're not followed, let's not break things further :)

Reply:
Ok, I will remove the commit.